### ISSUE TITLE
SMW::off SMW::on around meeting ref

### ISSUE THREAD

**Issue by olsonjaredm** [FodWikis/FodWikis#3]
Red links to SMW::off and SMW::on appear around meeting references footer on ROBO and EVA wikis.

![image](https://cloud.githubusercontent.com/assets/13458815/8918164/fda61802-347b-11e5-89f9-00221cd56ec4.png)

Examples:
https://mod2.jsc.nasa.gov/wiki/ROBO/index.php?title=JOTI
https://mod2.jsc.nasa.gov/wiki/EVA/index.php?title=FGB_PDGF

**Comment by jamesmontalvo3** [comment]
Yeah...the SMW::on/off thing is super annoying, but we haven't figured out how to fix it yet. Some of them may be caused by Extension:HeaderFooter, but some of our mega queries (like our OCAD query) causes it, too.
